tests/Doctrine/Tests/ORM/Functional/Ticket/GH5763Test.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\ORM\Id\AssignedGenerator;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+class GH5763Test extends OrmFunctionalTestCase
+{
+    protected function setUp()
```

Comment:
Let's be FW-compatible with phpunit 8
```suggestion
    protected function setUp() : void
```